### DISCUSSION TITLE
Welcoming @andersonhc as another maintainer for fpdf2!

### DISCUSSION THREAD

**Discussion by Lucas-C** [py-pdf/fpdf2#896] [Announcements]
Hi fpdf2 users!

@andersonhc has been a very active contributor of `fpdf2`.
Among other things:
* he implemented PDF encryption: https://github.com/py-pdf/fpdf2/pull/609
* he added the ability to have font fallbacks: https://github.com/py-pdf/fpdf2/pull/669
* he implemented text shaping with harfbuzz: https://github.com/py-pdf/fpdf2/discussions/696 & https://github.com/py-pdf/fpdf2/pull/820

He also actively took part in the resolution of several issues:
https://github.com/py-pdf/fpdf2/issues?q=mentions%3Aandersonhc+is%3Aissue

For all those reasons, and given his excellent code quality and his attention to details,
I invited him to become another maintainer for the `fpdf2` project.
He is now effectively an administrator of this GitHub repository.

Welcome @andersonhc! 😊👍🎉👏🤝

**Comment by gmischler** [comment]
A brillant choice! Congratulations! 🍾

Anderson has added very crucial and rather tricky functionality to the back-end of fpdf2 that others (including me) didn't dare to touch (even though we were talking about it a lot... :wink:).

I'm sure there's lots more of that to expect in the future.
Welcome to the team! 👍
